Firestore/Example/Tests/Integration/API/FIRDatabaseTests.m

```diff
@@ -21,6 +21,7 @@
 #import "Firestore/Example/Tests/Util/FSTIntegrationTestCase.h"
 #import "Firestore/Source/API/FIRFirestore+Internal.h"
 #import "Firestore/Source/Core/FSTFirestoreClient.h"
+#import "FIRGetOptions.h"
```

Comment:
You should not need to import this as it's part of FirebaseFirestore.

Reply:
Done. (Was unaware of the "umbrella" header that Gil pointed out below.)